midi_app_controller/gui/_tests/test_midi_status.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+
+from midi_app_controller.config import Config
+import os
+from midi_app_controller.models.controller import Controller
+from midi_app_controller.models.binds import Binds
+from PyQt5.QtCore import Qt
+from unittest.mock import MagicMock, patch
+from midi_app_controller.state.state_manager import SelectedItem
+
+
+@pytest.fixture
+def patch_rtmidi():
```

Comment:
Maybe it would be better to make whole `StateManager` a mock? And then just verify in tests that correct methods were called with correct arguments? 

`StateManager` has its own tests so there is no need to test it again with such a complicated half-mocked fixture.

If you decide to do so, it may be a good idea to make the `StateManager` singleton lazy (to make the mocking easier). For example by adding a `get_state_manager()` which will initialize it when it's called for the first time.